src/mongocxx/options/aggregate.cpp

```diff
@@ -64,6 +68,11 @@ aggregate& aggregate::collation(bsoncxx::document::view_or_value collation) {
     return *this;
 }
 
+aggregate& aggregate::let(bsoncxx::document::view_or_value let) {
+    _let = std::move(let);
```

Comment:
Since "let" has been passed by value, you've already got a copy of it. "_let = let" is probably fine. (But, what you've written won't hurt: move() is a fancy cast, it doesn't actually "move" anything on its own.)